package.json

```diff
@@ -40,6 +40,7 @@
     "@babel/core": "^7.4.3",
     "@babel/runtime": "^7.4.3",
     "@react-native-community/eslint-config": "0.0.5",
+    "@types/geojson": "^7946.0.7",
```

Comment:
It needs to be in normal `dependencies`, otherwise dependents would still need to install it